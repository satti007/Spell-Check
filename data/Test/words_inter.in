acheivement
admiting
amatuer
anwser
apropriate
aukward
benificial
beleive
chruch
contanct
coud
dessapoint
greatful
incredable
indefinate
jeprodise
laguage
leeiissy
letf
meenwhal
nessecary
ocassionaly
peice
playwrite
recieve
rember
revelent
sieze
sigth
suprise
suprised
surounded
swearinge
sylables
therefor
tierd
tommorow
tounge
vouture
sprinkel
