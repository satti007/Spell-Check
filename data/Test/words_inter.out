acheivement	achievement		
admiting	admitting		
amatuer	amateur		
anwser	answer		
apropriate	appropriate		
aukward	awkward		
benificial	beneficial		
beleive	believe		
chruch	church	crunch	churn
contanct	contact	content	contract
coud	cloud		
dessapoint	disappoint		
greatful	grateful	great	
incredable	incredible		
indefinate	indefinite		
jeprodise	jeopardize	jeopardise	
laguage language	luggage	
leeiissy	lazy		
letf	left	let	
meenwhal	meanwhile		
nessecary	necessary		
ocassionaly	occasionally		
peice	piece		
playwrite	playwright	playwriter	
recieve	receive		
rember	member	remember	
revelent	reverent	relevant	
sieze	seize	size	
sigth	sight	sigh	
suprise	surprise		
suprised	surprised		
surounded	surrounded		
swearinge	swearing	swearings	
sylables	syllables	syllabus	
therefor	therefore		
tierd	tired	tiered	tried
tommorow	tomorrow		
tounge	tongue	lounge	tough
vouture	vulture	voucher	
sprinkel	sprinkle		
